dandiapi/api/tasks.py

```diff
@@ -161,3 +163,12 @@ def validate_version_metadata(version_id: int) -> None:
 @shared_task
 def delete_doi_task(doi: str) -> None:
     delete_doi(doi)
+
+
+@shared_task
+def send_pending_users_email() -> None:
+    """Send an email to admins listing users with status set to PENDING"""
+    pending_users = User.objects.select_related('metadata').filter(
```

Comment:
IIRC `select_related` is only useful if you're going to use `user.metadata` somewhere, as it saves the DB from performing a separate `SELECT`. I don't think you actually do that anywhere though, all I see are accesses of `user.username` and `user.date_joined` in the template. Are those aliases for metadata fields?

Reply:
Oops, I think I was accessing `user.metadata` somewhere but then removed it. And `username` and `date_joined` are both fields directly on the `User` model, so yeah select_related isn't needed here